app/models/katello/ping.rb

```diff
@@ -226,10 +226,11 @@ def fetch_proxy(capsule_id)
       end
 
       def backend_status(url, backend)
-        ca_file = SETTINGS[:katello][backend][:ca_cert_file]
         options = {}
-        options[:ssl_ca_file] = ca_file unless ca_file.nil?
-        options[:verify_ssl] = SETTINGS[:katello][backend][:verify_ssl] if SETTINGS[:katello][backend].key?(:verify_ssl)
+        if (ca_file = ::Cert::Certs.backend_ca_cert_file(backend))
+          options[:ssl_ca_file] = ca_file
```

Comment:
Was this removed because we don't ever set the `verify_ssl` option for any of the backends?

Reply:
Indeed. It didn't make any sense to have separate logic for ping when the actual resources don't respect the setting.